pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -362,7 +362,7 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(ctx context.Context,
 	vmOwnerRefExists := false
 	if len(instance.OwnerReferences) != 0 {
 		for _, ownerRef := range instance.OwnerReferences {
-			if ownerRef.Kind == reflect.TypeOf(vmoperatortypes.VirtualMachine{}).Name() &&
+			if ownerRef.Kind == reflect.TypeOf(vmoperatorv1alpha1.VirtualMachine{}).Name() &&
```

Comment:
can we keep type as vmoperatorv1alpha3 instead of vmoperatorv1alpha1, like other places in this file ? or we specifically need vmoperatorv1alpha1 type here ?